control-plane/roles/monitoring/templates/grafana_dashboards/keycloak.yaml

```diff
@@ -0,0 +1,1261 @@
+---
```

Comment:
Not necessary, component is not part of the metal-stack. Can be cleaned up.

Reply:
https://github.com/metal-stack/metal-roles/pull/115/commits/f6f1982943779e34d7ef3f54a7ec9c5af3189312